public/javascript/ep-card-filter.js

```diff
@@ -194,9 +199,45 @@ $(document).ready(function(){
       window.cards.saveState();
     });
 
-    $('[data-section-toggle]').on('click', function(){
-      var section = $(this).attr('data-section-toggle');
+    $('.js-show-facet').on('change', function(){
```

Comment:
This line can be simplified by using `.change(handler)`, which is a [shortcut for `.on('change', handler)`](https://api.jquery.com/change/).